site/src/site/sphinx/en/release-notes.md

```diff
@@ -0,0 +1,123 @@
+Release Notes
+=============
+
+### :tada: :tada: [v3.0.4](https://github.com/alibaba/arthas/milestone/1?closed=1)  :exclamation:
+
+v2017-11-03
+----
+
+* [improvement] add [`getstatic`](getstatic.md)
+* [bug] fix Arthas class loader logs loading issues
+* [improvement] introduce [OGNL](https://en.wikipedia.org/wiki/OGNL) to customize `classloader` to invoke static methods
+* [improvement] optimise `termd` uppercase output performance 
+* [improvement] `classloader` compile in class loader category by default
+* [bug] fix `wc` counting issue
+* [improvement] disable certain JDK classes e.g. `Classloader`, `Method`, `Integer` and the lik
+* [improvement] quit directly when encountering incorrect [OGNL](https://en.wikipedia.org/wiki/OGNL) expression
+* [bug] fix `pipe` issues
+* [improvement] optimize command re-direct features using asynchronous log
+* [improvement] [`trace`](trace.md) can filter JDK method calls
+
+v2017-09-22
+----
+
+* [improvement] improve the error message when starting agent and server fails
+* [bug] fix some asynchronous issues
+
+v2017-09-11
+----
+
+* [improvement] [`async`](async.md) supported
+* [improvement] optimize [`jad`](jad.md) support JDK 8 and inner class
+* [bug] fix Chinese encoding issues
+
+v2017-05-11
+----
+
+* [improvement] [`tt`](tt.md) investigating/recording level one to avoid too much performance overhea
+* [bug] fix Chinese characters can not be presented issue
+
+v2017-05-12
+----
+
+* Arthas 3.0 release :confetti_ball:
+
+v2016-12-09
+----
+
+* [feature] [`as.sh`](https://github.com/alibaba/arthas/blob/master/bin/as.sh) support `-h` to print help info
+* [bug] [#121] fix leftover temp files causing Arthas cannot start issue
+* [bug] [#123] fix `attach/shutdown` repeatedly causing Arthas classloader leakage issue
+* [improvement] make the help info more readable
+* [bug] [#126] fix the documents links issues 
+* [bug] [#122] fix the [`classloader`](classloader.md) filtering out `sun.reflect.DelegatingClassLoader` issue
+* [bug] [#129] fix [`classloader`](classloader.md) presenting structure issues
+* [improvement]  [#125] make the Arthas log output more readable
+* [improvement]  [#96] [`sc`](sc.md) and more commands are supporting format as `com/taobao/xxx/TestClass`
+* [bug] [#124] fix the negative values of [`trace`](trace.md)
+* [improvement]  [#128] the output of [`tt`](tt.md) will auto-expand now
+* [bug] [#130] providing more meaningful error messages when port conflicts
+* [bug] [#98] fix Arthas starting issue: when updating/downloading failed, Arthas will fail to start
+* [bug] [#139] fix agent attaching fails under some scenarios issues
+* [improvement]  [#156] delay `jd-core-java` initialization to avoid Arthas starting failure
+* [bug] avoid thread names duplicate issue
+* [improvement]  [#150] filtering by total time cost in [`trace`](trace.md) 
+* [bug] fix [`sc`](sc.md) `NPE issue when searching `SystemClassloader` 
```

Comment:
The format looks malformed starting from here.

Reply:
`NPE issue when searching`  I think a '`' missing here.

Reply:
Many thanks, 👍 too many links and `` ` `` there. Fixed by `push -f`ed.